run-platform.sh

```diff
@@ -268,9 +268,26 @@ build_services() {
   fi
 }
 
+prepare_for_first_setup() {
+  pushd ${script_dir}/docker 1>/dev/null
+
+  echo -e "$blue_text""Preparing for initial setup""$default_text"
+  VERSION=$opt_version $docker_compose_cmd up -d db redis
+
+  echo -e "Creating a schema for Unstract in the database"
+  VERSION=$opt_version $docker_compose_cmd run backend .venv/bin/python manage.py create_schema
+  # TODO: Run migrations here once its removed from backend's entrypoint
+
+  popd 1>/dev/null
+}
+
 run_services() {
   pushd ${script_dir}/docker 1>/dev/null
 
+  if [ "$first_setup" = true ]; then
+    prepare_for_first_setup
+  fi
+
```

Comment:
@chandrasekharan-zipstack Let's move out backend schema creation from here.

Instead, call `create_backend_schema()` directly after `build_services` and before `run_services` below.

Also inside `create_backend_schema()` return early if `first_setup` is `false`.